app-eselect/eselect-wxwidgets/eselect-wxwidgets-20180529-r1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
(I guess this could be 8 but... not so important in my opinion)

Reply:
This ebuild should be going away before merge anyway, so I didn't bump